plugins/producer_plugin/include/eosio/producer_plugin/block_timing_util.hpp

```diff
@@ -42,28 +42,28 @@ namespace block_timing_util {
    // In the past, a producer would always start a block `config::block_interval_us` ahead of its block time. However,
    // it causes the last block in a block production round being released too late for the next producer to have
    // received it and start producing on schedule. To mitigate the problem, we leave no time gap in block producing. For
-   // example, given block_interval=500 ms and cpu effort=400 ms, assuming the our round start at time point 0; in the
+   // example, given block_interval=500 ms and cpu effort=400 ms, assuming our round starts at time point 0; in the
    // past, the block start time points would be at time point -500, 0, 500, 1000, 1500, 2000 ....  With this new
    // approach, the block time points would become -500, -100, 300, 700, 1100 ...
-   inline fc::time_point production_round_block_start_time(uint32_t cpu_effort_us, chain::block_timestamp_type block_time) {
+   inline fc::time_point production_round_block_start_time(fc::microseconds cpu_effort_us, chain::block_timestamp_type block_time) {
```

Comment:
maybe we can rename the variable `cpu_effort` now that the type enforces microseconds?